### PR TITLE
Elasticsearch.Net 7.0.0 support, V7 templates

### PR DESCRIPTION
**Pull request by kanadaj**
**What issue does this PR address?**
https://github.com/serilog/serilog-sinks-elasticsearch/issues/248 targeting Elasticsearch.Net 7.0.0-alpha2 on .NET Standard 2.0; still targeting 6.0.0 on .NET Standard 1.1 and .NET Framework 4.5.1 since 7.0.0 is not compatible.

A small note: why target .NET Framework separately?

**Does this PR introduce a breaking change?**
Possibly. I cannot test against ES 6 right now so would need verification whether the 7.0.0 SDK of ElasticSearch breaks it or not.

**Other information**:
Needs testing against ES 6.X.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
     "sdk": {
-        "version": "2.1.4"
+        "version": "2.1.500"
```
Comment (ffMathy) [inline global.json]: Why 500 here?

Comment (kanadaj) [reply global.json]: @ffMathy oh that's just from my initial setup cause I don't have 2.1.4 locally and when I did that I was just experimenting, didn't consider I'd end up making a PR...

Comment (ffMathy) [reply global.json]: Ah I see.

**Comment by dominiclapointe** [comment]
Is there a reason that it has not been merged yet? Really looking forward for the next nuget package that is compatible with esv7!!

**Comment by kanadaj** [comment]
@dominiclapointe Yes, the fact that this pull request also contains a dependency on the Preview version of the SDK itself, and causes some other complications like leaving the dependency of the netcore1.3 package as 6.X and netcore2.0 as 7.X - they kind of have to figure out what to do with netcore1.3.

Also, I'm still not sure the preview SDK for 7.X is compatible at all with ES6

**Comment by naymore** [comment]
Does anybody know what Elastic's idea is with its dotnet SDK?  
Is it backward compatible?  
Will 6.8 receive future updates (hotfix releases like 6.8.1, ..)? (In the past this almost never happened)  
Does the ES SDK for dotnet version match the supported ES version?  

ES v6 will be supported for quite some time, [see EOL](https://www.elastic.co/support/eol), however v7 introduced breaking changes and [v8 will introduce even more](https://www.elastic.co/guide/en/elasticsearch/reference/master/breaking-changes-8.0.html).

btw, 7.0.0-beta1 has been released ;-)
Maybe this sink should also reflect the ES SDK used in its version scheme.
v7.1.0 sink only supports v6.8 ES SDK :|

It might be simpler to have ES-version-specific sinks (at least for ES v6.x and v7+) rather than having one sink for any ES version and also be in doubt what's even supported or breaking :)

**Comment by mivano** [comment]
@Mpdreamz can you give some feedback here about the plans?

**Comment by NeilCross** [comment]
Elasticsearch.net 7.0.0 has been released on nuget recently: https://www.nuget.org/packages/Elasticsearch.Net/

**Comment by Mpdreamz** [comment]
I've started on a continuation of this PR here:

**Pull request merged**